staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -43,6 +43,14 @@ const (
 	logTimestampFormat     = "2006-01-02T15:04:05.000000Z"
 )
 
+var noisyLibvirtErrors = []string{
+	"internal error: Unable to get system bus connection: Could not connect: No such file or directory",
+	`internal error: Unable to get session bus connection: Cannot spawn a message bus without a machine-id: Unable to load /var/lib/dbus/machine-id or /etc/machine-id: Failed to open file`,
+	"Unable to open /dev/kvm: Permission denied",
+	"Client hit max requests limit 5. This may result in keep-alive timeouts. Consider tuning the max_client_requests server parameter",
```

Comment:
Sorry for not noticing this earlier, but do you have an idea why we have more than 5 clients to libvirt?
libvirt's request to consider a higher max_client_requests makes sense. Could you leave this error out and file an issue to solve it properly?

Reply:
We have multiple agent poller workers running simultaneously - this is the main reason i believe

sure will remove it, but imo it mislead us a lot already, so better to keep it V(3) and open an issue in parallel,
anyhow whatever you prefer

opened issue https://github.com/kubevirt/kubevirt/issues/16276